plugin/build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
 plugins {
     id 'java-gradle-plugin'
     id 'maven-publish'
-    id 'com.gradle.plugin-publish' version '0.12.0'
+    id 'com.gradle.plugin-publish' version '0.21.0'
```

Comment:
It is better to update to 1.0+ version which has fixed a lot of issue in plugin publishing - it should now natively work with `maven-publish` and `signing` plugins

Reply:
The 1.0 version introduced too many changes. I would upgrade to it in a separate effort.